src/base/matrix.rs

```diff
@@ -2358,6 +2358,2366 @@ impl<T> super::alias::Matrix1<T> {
     }
 }
 
```

Comment:
The following lines contain the open source duplicated code. Only last 21 lines with into_vec method represents my contribution.